README.md

```diff
@@ -40,4 +40,4 @@ for client libraries to work.
 ## How to run the application tests
 
 1. Set up the emulator as described in #1 above.
```

Comment:
At some point we'd want to bring up the emulator as part of `mvn integration-test` right ?

Reply:
In the examples I've seen so far this is done before maven, e.g: https://github.com/googleapis/java-spanner-jdbc/blob/master/.github/workflows/integration-tests-against-emulator.yaml

It would probably be easier to follow this pattern

Reply:
I see. Yeah I guess as long as we don't have to manually run all the commands to bring up the emulator and set up the env variables etc., we should be good.

Reply:
Currently, we do run under the assumption that the emulator is being run in the background, which seems to be what the [java-docs-samples Spanner JDBC repository does](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/spanner/jdbc/src/test/java/com/example/spanner/jdbc/JdbcExamplesIT.java). It would be ideal to set up some sort of way to have the emulator start up with a command or something, but for the sake of time, I don't think this should be included in this PR.

Reply:
https://github.com/GoogleCloudPlatform/cloud-spanner-samples/issues/31

Reply:
Acknowledged. SG, thanks!